lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -108,17 +108,33 @@ func init() {
 }
 
 // Client is the RDP client.
+// It's lifecycle is:
+//
+// ```
+// rdpc := New() // creates client and kicks off RDP connection
+// rdpc.Wait()   // waits for the duration of the connection
+// ```
+//
+// rdpc.Wait() typically takes care of calling rdpc.Cleanup().
+// However, if something fails between the call to New() and
+// rdpc.Wait(), the caller MUST call rdpc.Cleanup() to ensure
+// all memory is released.
 type Client struct {
 	cfg Config
 
 	// Parameters read from the TDP stream.
 	clientWidth, clientHeight uint16
 	username                  string
 
-	// handle allows the rust code to call back into the client
+	// handle allows the rust code to call back into the client.
+	// Once it is created with cgo.NewHandle(), the original caller
+	// MUST release it with a call to handle.Delete().
 	handle cgo.Handle
 
 	// RDP client on the Rust side.
+	// Once it is created by assigning this field to the result of C.connect_rdp(),
+	// the caller SHOULD end the RDP connection by passing it to C.close_rdp(),
+	// and the caller MUST free its memory by passing it to C.free_rdp().
```

Comment:
These comments are so detailed that they are both:

- hard for someone not familiar with the internal details to understand, and
- easy to go out of date in the future

Can we get rid of them?